sdk/test-utils/perf/CHANGELOG.md

```diff
@@ -2,6 +2,18 @@
 
 ## 1.0.0 (Unreleased)
 
+### 2021-11-24
+
+- Adds `PerfTestBase` and `BatchPerfTest`.
+  - `PerfTestBase` abstract class with the skeleton of the perf tests.
+  - `BatchPerfTest` extends `PerfTestBase`, `BatchPerfTest` enables writing perf tests with more flexibility where the number of operations are dynamic for the method/call being tested.
+  - `PerfTest` now extends `BatchPerfTest`, a single call in the "run" method for the method being tested counts as one operation - has no breaking changes now and is supposed to work as it did before.
```

Comment:
nit

```suggestion
  - `PerfTest` now extends `BatchPerfTest`. Going forward, a single call in the "run" method for the method being tested counts as one operation. This is not a breaking change, so existing performance tests will continue to work without changes.
```

Reply:
> Going forward, a single call in the "run" method for the method being tested counts as one operation.

Not just going forward, it has always been the same.

Updating to 
> - `PerfTest` now extends `BatchPerfTest`. A single call in the "run" method for the method being tested counts as one operation. `PerfTest` related updates are not breaking, so existing performance tests will continue to work without changes.

Reply:
```suggestion
  - `PerfTest` now extends `BatchPerfTest`. A single call in the "run" method for the method being tested counts as one operation. `PerfTest` related updates are not breaking, so existing performance tests will continue to work without changes.
```